src/main/java/com/harragan/battleshipsboot/controllers/ShipController.kt

```diff
@@ -0,0 +1,63 @@
+package com.harragan.battleshipsboot.controllers
+
+import com.harragan.battleshipsboot.facades.ShipPlacingFacade
+import com.harragan.battleshipsboot.model.ships.*
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+
+@RestController
+class ShipController {
```

Comment:
could move to "/place" and then have a when statement that selects which ship to place, might be favorable if all of these endpoints share the same request body